### ISSUE TITLE
Notification - WCAG rule 1.4.6

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/grommet#7604]
**Failed rule**

1.4.6
Contrast (Enhanced)
Level AAA

The visual presentation of text and images of text has a contrast ratio of at least 7:1

---

**Notes**

This is a Level AAA contrast fail.

**Comment by jcfilben** [comment]
Since this is AAA and we are unlikely to alter the colors for the theme beyond AA requirements I'm going to mark this issue as 'not planned'
